packages/wallets/blocto/src/adapter.ts

```diff
@@ -157,4 +157,14 @@ export class BloctoWalletAdapter extends BaseWalletAdapter {
             throw error;
         }
     }
+
+    async sendVersionedTransaction(
```

Comment:
Shouldn't be needed if we protect this method. And it's possible if we fix Blocto's `supportedTransactionVersions` to null, the type of `sendTransaction` shouldn't need to change either.

Reply:
> And it's possible if we fix Blocto's `supportedTransactionVersions` to null, the type of `sendTransaction` shouldn't need to change either.

Can you elaborate more here on why the type of `sendTransaction` shouldn't need to change?

Reply:
This is a guess, but if https://github.com/solana-labs/wallet-adapter/pull/558#discussion_r969010739 is implemented, `sendTransaction` should be statically known to only accept a `Transaction`.